pkg/core/interop/context.go

```diff
@@ -236,7 +236,7 @@ func (c *ContractMD) GetMethodByOffset(offset int) (MethodAndPrice, bool) {
 func (c *ContractMD) GetMethod(name string, paramCount int) (MethodAndPrice, bool) {
 	index := sort.Search(len(c.Methods), func(i int) bool {
 		md := c.Methods[i]
-		res := strings.Compare(name, md.MD.Name)
+		res := strings.Compare(strings.ToLower(name), strings.ToLower(md.MD.Name))
```

Comment:
Will affect every call, meh... OK, whatever, let's make it compatible for now.

Reply:
It is used for Oracle contract only, so it's not critical.